querybook/webapp/components/StatementResultTable/StatementResultTable.tsx

```diff
@@ -109,6 +125,23 @@ export const StatementResultTable: React.FunctionComponent<{
         filterConditionByColumn,
     } = useFilterCell(rows);
 
+    const [isDisplayFull, setIsDisplayFull] = useState(false);
+
+    const visibilityToggle = useCallback(() => {
+        const columnNames = data[0];
```

Comment:
you can nest setExpandedColumn in setIsDisplayFull so you don't have to have isDisplayFull as a dependency:

```
setIsDisplayFull((old) => {
    const newDisplayFull = !old;
    setExpandedColumn(...)
```